ci/jenkins/test-vmc.sh

```diff
@@ -255,8 +271,14 @@ function deliver_antrea {
     done
 
     kubectl get nodes -o wide --no-headers=true | awk '{print $6}' | while read IP; do
-        rsync -avr --progress --inplace -e "ssh -q -o StrictHostKeyChecking=no -i ${GIT_CHECKOUT_DIR}/jenkins/key/antrea-ci-key" antrea-ubuntu.tar capv@${IP}:/home/capv/antrea-ubuntu.tar
-        ssh -q -o StrictHostKeyChecking=no -i ${GIT_CHECKOUT_DIR}/jenkins/key/antrea-ci-key -n capv@${IP} "sudo crictl images | grep 'antrea-ubuntu' | awk '{print \$3}' | xargs -r crictl rmi ; sudo ctr -n=k8s.io images import /home/capv/antrea-ubuntu.tar ; sudo ctr -n=k8s.io images tag docker.io/antrea/antrea-ubuntu:${DOCKER_IMG_VERSION} docker.io/antrea/antrea-ubuntu:latest ; sudo crictl images | grep '<none>' | awk '{print \$3}' | xargs -r crictl rmi" || true
+        ssh-keygen -f "/var/lib/jenkins/.ssh/known_hosts" -R ${IP}
+        scp -o StrictHostKeyChecking=no -i ${GIT_CHECKOUT_DIR}/jenkins/key/antrea-ci-key antrea-ubuntu.tar capv@${IP}:/home/capv
+        if [ $TEST_OS == 'centos-7' ]; then
+            ssh -o StrictHostKeyChecking=no -i ${GIT_CHECKOUT_DIR}/jenkins/key/antrea-ci-key -n capv@${IP} "sudo chmod 777 /run/containerd/containerd.sock"
```

Comment:
Do you think we can ignore message with `-q` here and below?

Reply:
`-q` added.